src/client/components/TabPaneContent2/index.js

```diff
@@ -1,73 +1,76 @@
+import { Trans, useTranslation } from "react-i18next";
 import BPOButton from "../BPOButton";
 import React from "react";
-// import { useTranslation } from "react-i18next";
 
 function TabPaneContent2() {
-  // const { t } = useTranslation();
+  const { t } = useTranslation();
 
   return (
     <div>
-      <h3>1. Register with Benefit Programs Online</h3>
+      <h3>
+        <Trans t={t} i18nKey="tab2.header10" />
+      </h3>
       <p>
-        <strong>
-          The fastest way to apply for all unemployment benefits is through UI
-          Online. After you have registered for a Benefit Programs Online
-          account, you can get started on UI Online.
-        </strong>{" "}
-        You can still apply for UI by{" "}
-        <a href="https://edd.ca.gov/Unemployment/Filing_a_Claim.htm">
-          phone, mail, or fax
-        </a>
-        .
+        <Trans t={t} i18nKey="tab2.p10">
+          {
+            // The content of this <Trans> must match translation.json
+            // otherwise the link won't render
```

Comment:
Should we set debug=true on Trans tags where we are matching tags? It looks like this will cause some JS console spew so it's easier to notice when this happens. Maybe only debug=true in non-prop builds?

Reply:
I've set debug: true in i18n.js elsewhere in this PR, does that cover what you're asking for? if not can you explain more what you'd like to happen?

Reply:
Oh, I think that's the same thing! Yes, that sounds good.